pkg/models/base_task_list_section.go

```diff
@@ -38,7 +38,7 @@ func NewBaseTaskListSection(createdBy string, modelPlanID uuid.UUID) baseTaskLis
 
 }
 
-//GetBaseTaskListSection returns the BaseTaskListSection Object embedded in the struct
+// GetBaseTaskListSection returns the BaseTaskListSection Object embedded in the struct
```

Comment:
I think my editor went through and automatically changed all these comments. I can move them to a different PR if we don't want them in there.

Reply:
>  It's fine with me to leave them here since they are already here.